atlas_placement_hints/app/placement_hints.py

```diff
@@ -305,41 +311,98 @@ def isocortex(
     required=True,
     help="path of the directory to write. It will be created if it doesn't exist.",
 )
+@click.option(
+    "--thalamus-meshes-dir",
+    required=True,
+    help="""Path of the directory to use for either saving or loading thalamus
+    meshes. It will be created if it doesn't exist.""",
+)
+@click.option(
+    "--create-uncut-thalamus-meshes-flag",
```

Comment:
you don't need to put `-flag` at the end, its redundant
you can use a single option, `--load-meshes`  (we know its thalamus, right?) and if it is not present, you create them (default behaviour?) If one if False, the other must be True, and vice-versa, no? So a single one works?